be/src/common/config.h

```diff
@@ -959,4 +959,7 @@ CONF_mInt64(load_tablet_timeout_seconds, "30");
 
 CONF_mBool(enable_pk_value_column_zonemap, "true");
 
+// Used by default mv resource group
+CONF_mDouble(default_mv_resource_group_memory_limit, "0.8");
```

Comment:
why they are set to 0.8 and 1 for mem and cpu?

Reply:
cpu:
In actual business scenarios, CPU cores that are allocated to the resource group proportionally scale based on the availability of CPU cores on the BE.
If cluster is too busy, default mv resource group should not assign too much core. 
If cluser is idle, cpu is enough to mv task run.

memory:
Default mv resource group is designed to avoid that case when user do not know the concept of resource group—— Cluster resource fully allocated to mv task so that query and load can not work. So we need to leave some memory to keep cluter work normal . So memory config can not be to high.

When user do not know the concept of resource group, There is also another case   ——
User already have a lot of mv in cluster in lower version which not contain default mv resource group , after upgrade starrokcs, they may find a lot of mv task failed because of memory limit. They have to learn the concept of resource group,  change config and restart cluster to let it work. So memory config can not be too low.